sycl/test/host-interop-task/host-task.cpp

```diff
@@ -15,9 +15,22 @@
 // RUNx: %GPU_RUN_PLACEHOLDER %t.out 4
 // RUNx: %ACC_RUN_PLACEHOLDER %t.out 4
 
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out 5
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 5
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 5
+
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out 6
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 6
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 6
+
+// RUNx: %CPU_RUN_PLACEHOLDER %t.out 7
+// RUNx: %GPU_RUN_PLACEHOLDER %t.out 7
+// RUNx: %ACC_RUN_PLACEHOLDER %t.out 7
```

Comment:
Why are all these test cases disabled?

Reply:
The tests are added in disabled state due to existing issue which will lead to hang.
The follow-up patch with issue fix will enable these tests.
I've updated patch description to note that explicitly.